vendor/opam-file-format/src/dune

```diff
@@ -1,6 +1,20 @@
 (library
- (name opam_file_format)
- (wrapped false))
+  (name opam_file_format)
+  (synopsis "Parser and printer for the opam file syntax")
+  (wrapped false)
+  (modules :standard \ flags)
+  (flags :standard (:include flags.sexp)))
+
+(rule
+  (with-stdout-to flags.ml
+    (echo "print_string (if String.sub Sys.ocaml_version 0 5 = \"4.02.\" then \"(-w -50)\" else \"()\")")))
```

Comment:
We don't support 4.02 in dune so this isn't needed

Reply:
Ok, removed that bit and the whole part about flags.